### ISSUE TITLE
"Fork me on Github" can cover "browse packages" link making latter unclickable

### ISSUE THREAD

**Issue by ddrone** [commercialhaskell/stackage-server#119]
To reproduce, try reducing browser window to a certain point or opening the website on a smartphone.

**Comment by chrisdone** [comment]
Just needs a responsive bit in the CSS.

**Comment by Nicowcow** [comment]
I agree, but at the same time it is very handy, as I knew where to click to report the issue : ) Could it be place on the left side, which doesn't have as much content?

**Comment by chrisdone** [comment]
Sure, the left works. PR?
